src/System.Runtime.Serialization.Formatters/tests/SerializationTypes.cs

```diff
@@ -115,7 +135,8 @@ public Tree(T value, Tree<T> left, Tree<T> right)
         public override bool Equals(object obj)
         {
             Tree<T> o = obj as Tree<T>;
-            if (o == null) return false;
+            if (o == null)
```

Comment:
Don't go back now, but usually FYI for ask mode fixes we miniimize diffs, especially reformatting, to focus attention on the functional changes.